src/util/orgUnits.js

```diff
@@ -49,7 +49,9 @@ export const fetchOrgUnitGroupSet = (id) =>
         `/organisationUnitGroupSets/${id}?fields=organisationUnitGroups[id,name,color,symbol]`
     ).then(parseGroupSet)
 
-export const filterPointFacilities = (data) => data.filter((d) => d.ty === 1)
+export const filterPoints = (data) => data.filter((d) => d.ty === 1)
```

Comment:
`filterPoints` (and `filterPolygons`) are kind of confusing names as it's unclear if filterPoints is filtering in point, or filtering out points.  What about `getPointItems` or `getPointGeometries` or similar?

Reply:
Changed to `getPointItems` and `getPolygonItems`